sky/skylet/skylet.py

```diff
@@ -11,6 +11,7 @@
 EVENTS = [
     events.JobUpdateEvent(),
     events.AutostopEvent(),
+    events.SpotJobUpdateEvent(),
```

Comment:
nit: order these?

Reply:
Done. And added a comment for the order of SpotJobUpdateEvent vs JobUpdateEvent.